library/src/main/java/com/raizlabs/android/dbflow/sql/SqlUtils.java

```diff
@@ -409,7 +409,10 @@ void delete(final TableClass model, AdapterClass modelAdapter, boolean async) {
      * @param table  The table of the model
      */
     public static void notifyModelChanged(Class<? extends Model> table, BaseModel.Action action) {
-        FlowManager.getContext().getContentResolver().notifyChange(getNotificationUri(table, action), null, true);
+        SQLiteDatabase db = FlowManager.getDatabaseForTable(table).getWritableDatabase();
+        if (!db.inTransaction()) {
```

Comment:
Ok So this became a lot more _interesting_ of an issue. i will have to introduce in `2.0.0` a new way to notify when models change outside of transactions.